test/bntest.c

```diff
@@ -2208,10 +2044,15 @@ static int file_test_run(STANZA *s)
     const FILETEST *tp = filetests;
 
     for ( ; --numtests >= 0; tp++) {
-        if (findattr(s, tp->name) != NULL)
-            return tp->func(s);
+        if (findattr(s, tp->name) != NULL) {
+            if (!tp->func(s)) {
+                TEST_info("Failed %s test at %d", tp->name, s->start);
+                return 0;
+            }
+            return 1;
+        }
     }
-    fprintf(stderr, "Unknown test at %d\n", s->start);
+    TEST_info("Unknown test at %d", s->start);
```

Comment:
the caller will report a test failure, so the TEST_info calls in this routine just add more info.